docs/source/concept_models/vivarium_csu_multiple_myeloma_2/concept_model.rst

```diff
@@ -147,15 +141,35 @@ set of treatment categories based on treatment guidelines from the `NCCN
 5.0 Simulation Scenarios
 ++++++++++++++++++++++++
 
+We know that our scenarios will roughly correspond to:
+
+* Scenario 0: Naive treatment assignment
+* Scenario 1: baseline/low Isa uptake (e.g., 0% uptake of Isa following Dara frontline treatment)
+* Scenario 2: 5% uptake of Isa following Dara frontline treatment
+* Scenario 3: 5% uptake of Isa frontline treatment (e.g, instead of Dara frontline treatment, replacing a certain 'other' treatment regimen, etc.)
+
+.. todo::
+
+  Rigorously define these scenarios.
+
 6.0 Simulation Parameters
 +++++++++++++++++++++++++
 
 6.1 Locations
 -------------
 
+United States.
+
 6.2 Population and Randomness
 -----------------------------
 
+Population description:
+
+* Cohort type: Prospective closed cohort of individuals aged 15 years and older (open cohort if we only examine on age 50+ population)
+* Size of largest starting population: 100,000 simulants
```

Comment:
Note: Phase 1 concept model says 200k, but it looks like we were already using 100k by the time of our last Model 9 run.